feat(gatsby-plugin-preload-fonts): Add option to run in CI (#21037)
gatsbyjs/gatsby: packages/gatsby-plugin-preload-fonts/README.md

See [Puppeteer](#puppeteer) for more information.

### Using with CI

> If you're planning to use this plugin in CI you can set environment variable `CI`. Please note that in this case `font-preload-cache.json` will only get rebuild when routes change.

Reviewer: ```suggestion
If you're planning to use this plugin in CI you can set the environment variable `CI`. Please note that in this case `font-preload-cache.json` will only get rebuild when routes change.
```
I think it would be helpful to give an example of using the environment variable. Is it just a flag? Does it have a value?

Author: From what I've seen in the code isCI utility checks for name and if it doesn't match anything than it will accept anything. I guess that `CI=true` would be suitable for example. Please tell me what you think about it

Reviewer: That sounds great.

Author: Updated ;)

Reviewer 2: I'm pretty sure all CI environments will set the `CI` env var automatically

> If you're planning to use this plugin in CI you can set environment variable `CI=true`. Please note that in this case `font-preload-cache.json` will only get rebuild when routes change.

Reviewer: ```suggestion
If you're planning to use this plugin in CI you can set the environment variable `CI=true`. Please note that in this case `font-preload-cache.json` will only get rebuild when routes change.
```

> If you're planning to use this plugin in CI you can set environment variable `CI=true`. Please note that in this case `font-preload-cache.json` will only get rebuild when routes change.

Reviewer 3: ```suggestion
If you're planning to use this plugin in CI you can set the environment variable `CI=true` (most CI providers like Travis and CircleCI will set this for you). Please note that in this case `font-preload-cache.json` will only get rebuilt when routes change.
```
